src/line-graph.js

```diff
@@ -393,16 +395,43 @@ export class LineGraph extends HTMLElement {
               end: xScale.invert(extent[1])
             }
           }));
-    
+
           let getBrushArea = d3.select(this.shadowRoot.getElementById('brushArea'));
           getBrushArea.call(brush.move, null);
         }
       });
-      svg.append("g")
-      .attr("id","brushArea")
+    svg.append("g")
+      .attr("id", "brushArea")
       .attr("class", "brush")
-       .call(brush)
-  
+      .call(brush)
+
+  }
+
+  /**
+   * 
+   * @param d Data points object [value,time]
+   * @param ttlD An object with key unit and color and group
+   * @returns Tooltip html body
+   */
+   
+  tooltipTemplate(d,ttlD){
```

Comment:
Can we place this template in separate file? Perhaps `/templates/tooltip`

Reply:
Yeah, But charts don't need any HTML template anywhere except a tooltip.